### PR TITLE
[2181] Hook up course withdraw

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Withdrawing courses

### Changes proposed in this pull request
- Ensure withdrawing a course is "self-service"

### Guidance to review
Find a running course and click withdraw in the sidebar.

### CODE REVIEW

**Comment by dankmitchell** [comment]
[Hook up "course withdraw"](https://trello.com/c/3DCKPaJd/2181-hook-up-course-withdraw)

**Comment by fofr** [comment]
FYI, how a withdrawn course should look:
Screen Shot 2019-10-03 at 14 26 09

File: app/views/courses/_course_table_row.html.erb [hunk 1]
```diff
@@ -12,7 +12,8 @@
     </td>
   <% end %>
   <td class="govuk-table__cell" data-qa="courses-table__status">
-    <% unless course.not_running? %>
+    <!-- TODO: Test this -->
+    <% unless course.not_running? || course.is_withdrawn? %>
```
Comment (fofr) [inline app/views/courses/_course_table_row.html.erb]: Move this into the course model?

Comment (fofr) [reply app/views/courses/_course_table_row.html.erb]: Wherever `not_running?` is used, we might want to do the same for withdrawn

**Comment by defong** [comment]
really tempted to reject this as its red.... but its that linter thats moaning

File: app/views/courses/_course_table_row.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
     </td>
   <% end %>
   <td class="govuk-table__cell" data-qa="courses-table__status">
-    <% unless course.not_running? %>
+    <% if course.is_running? || course.is_withdrawn? %>
```
Comment (fofr) [inline app/views/courses/_course_table_row.html.erb]: When the work is done to update the manually withdrawn courses, we can probably do away with this `if` statement.

Comment (danielburnley) [reply app/views/courses/_course_table_row.html.erb]: :+1:

**Pull request merged**